pkgs/development/python-modules/jira/default.nix

```diff
@@ -1,29 +1,35 @@
 { lib, buildPythonPackage, fetchPypi, isPy3k
 , pytest, pytestrunner, pbr, glibcLocales , pytestcov
-, requests, requests_oauthlib, requests_toolbelt, defusedxml }:
+, requests, requests_oauthlib, requests_toolbelt, defusedxml
+, flaky, ipython
```

Comment:
I don't see where `flaky` is used below.

Reply:
ahh dammit, this was part of the attempt to get the test suite running :sweat_smile:

Reply:
... it's a lot easier to find this stuff in other people's code than in your own :smile: